src/clr-angular/accordion/_variables.accordion.scss

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+$clr-accordion-text-color: $clr-color-neutral-1000 !default;
```

Comment:
Any reason this wouldn't pull from our global text color?

Reply:
updated